sql/core/src/main/scala/org/apache/spark/sql/SQLConf.scala

```diff
@@ -30,6 +30,7 @@ private[spark] object SQLConf {
   val DEFAULT_SIZE_IN_BYTES = "spark.sql.defaultSizeInBytes"
   val SHUFFLE_PARTITIONS = "spark.sql.shuffle.partitions"
   val CODEGEN_ENABLED = "spark.sql.codegen"
+  val UNSAFE_ENABLED = "spark.sql.unsafe.enabled"
```

Comment:
unless we think there are going to be other options under unsafe,  we might consider getting rid of `enabled` to match `codegen`